WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1736,3 +1736,8 @@ CheckBoxTableCellSpacing    = 6px
 CheckboxTableLeftText       = text-align:right; margin-right:@CheckBoxTableCellSpacing@;
 CheckboxTableRightText      = text-align:left; margin-left:@CheckBoxTableCellSpacing@;
 
+###################
+#  Dialogs
```

Comment:
Can we move dialog related below property to the same section in the property file.
Preferably, DialogBodyText , DialogBody